tests/translator/input/error_graphqlapi_no_schema_inline_or_uri.yaml

```diff
@@ -0,0 +1,16 @@
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  SuperCoolAPI:
+    Type: AWS::Serverless::GraphQLApi
+    Properties:
+      SchemaInline: |
```

Comment:
nit: The file says `no_schema_inline_or_uri`, but the content is more like containing both`inline and uri`?

Reply:
Great catch! I think I accidentally changed the name when I was fixing the branch